src/distribution/gamma.rs

```diff
@@ -566,14 +564,14 @@ mod test {
         test_almost(10.0, 10.0, 1.0251532120868705806216092933926141802686541811003037e-30, 1e-44, |x| x.pdf(10.0));
         test_almost(10.0, 1.0, 0.0000010137771196302974029859010421116095333052555418644397, 1e-20, |x| x.pdf(1.0));
         test_almost(10.0, 1.0, 0.12511003572113329898476497894772544708420990097708601, 1e-15, |x| x.pdf(10.0));
-        test_is_nan(10.0, f64::INFINITY, |x| x.pdf(1.0));
-        test_is_nan(10.0, f64::INFINITY, |x| x.pdf(f64::INFINITY));
+        test_is_nan(10.0, f64::INFINITY, |x| x.pdf(1.0)); // is this really the behavior we want?
```

Comment:
Honestly not sure what the desired behavior is in this case. We can leave it for now and open another issue for discussion. See #57